Tests iOS/SettingsTests.swift

```diff
@@ -174,6 +189,15 @@ class SettingsTest: XCTestCase {
         XCTAssertFalse(app.deleteConfirmationView.deleteAccountButton.isEnabled)
     }
 
+    @MainActor
+    func tapSettings() async {
+        app.tabBar.settingsButton.wait().tap()
+//        XCTAssertTrue(app.settingsView.exists)
+
+        let settingsViewEvent = await snowplowMicro.getFirstEvent(with: "global-nav.settings")
+//        XCTAssertNotNil(settingsViewEvent)
```

Comment:
can we remove the commented out code?

Reply:
@bassrock should we keep these in or remove them? They were to test micro, but they were failing when we were testing yesterday.

Reply:
@timc-mozilla lets uncomment them I think it was your local setup